style.css

```diff
@@ -96,22 +99,270 @@ header {
+  text-align: center;
+  color: #344563;
+}
+
+.language {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  flex-direction: row;
+  text-align: center;
+  margin-bottom: 36px;
+  margin-top: 12px;
+  padding-left: 0;
+  gap: 12px;
+}
+
+.language li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  padding-left: 10px;
+  padding-right: 10px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.project {
+  margin-top: 114px;
+  gap: 50px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.button {
+  width: 124px;
+  height: 48px;
+  display: flex;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  margin-bottom: 28px;
+}
+
+.see-btndark {
+  padding: 12px;
+  border-radius: 8px;
+  color: #396df2;
+  display: inline-block;
+  font-weight: 500;
+  font-size: 17px;
+  text-decoration: none;
+}
+
+.about_section {
+  background-color: #fff;
+  border-top-right-radius: 50px;
+  padding-top: 8%;
+  margin-top: 12px;
+  width: 100%;
+}
+
+.me {
+  width: 327px;
+  height: 144px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  text-align: center;
+}
+
+button {
+  border: 1px solid #6070ff;
+  padding: 12px;
+  border-radius: 8px;
+  color: #396df2;
+  margin-left: 16px;
+  margin-bottom: 30px;
+  background-color: #fff;
+}
+
+.resume {
+  padding: 12px;
+  border-radius: 8px;
+  color: #396df2;
+  display: inline-block;
+  font-weight: 500;
+  font-size: 17px;
+  text-decoration: none;
+}
+
+.abt-heading {
+  margin: 0;
+  padding-top: 200px;
+  padding-left: 24px;
+  padding-right: 24px;
+  height: 150px;
+  width: 87.2%;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.abt-desc {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  align-items: center;
+  margin-bottom: 12px;
+  padding-left: 24px;
+  padding-right: 24px;
+  width: 87.2%;
+}
+
+.Language {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  margin-left: 24px;
+  list-style-type: none;
+}
+
+.fa-angle-down,
+.fa-angle-right {
+  padding-right: 64px;
   color: #505f79;
 }
+
+.icon-languages li {
+  display: flex;
+  align-items: center;
+  background: #f7f7f9;
+  border-radius: 8px;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+  margin: 16px;
+  gap: 16px;
+  padding: 12px;
+}
+
+.icon-languages {
+  padding-left: 0;
+}
```

Comment:
- [ ] Please I suggest wrapping the content of the `icon-languages` class with a CSS Flexbox. 😃